pyirf/interpolation/tests/test_visible_edges_extrapolator.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+import pytest
+
+
+def test_find_visible_facets():
+    """Test for visible facets finding utility"""
+    from pyirf.interpolation.visible_edges_extrapolator import find_visible_facets
+
+    grid = np.array([[0, 0], [20, 0], [40, 0], [10, 20], [30, 20], [10, 10]])
+
+    target1 = np.array([[0, -20]])
+    res1 = find_visible_facets(grid_points=grid, target_point=target1)
+
+    # From target1 facets spanned by grid point 0 to 1 and 1 to 2 are visible
+    assert np.all((res1 == grid[0]) | (res1 == grid[1]) | (res1 == grid[2]))
```

Comment:
If I read this correctly, this just tests that at least one of the expected points is in the result, e.g. the test would pass if I return `res1 = grid[0]`.

So it doesn't actually test the correct lines are returned.

Reply:
Yeah, that's correct. I've extended the tests to additionally check that all expected points are present and the shapes are what they should.